components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -592,6 +592,7 @@ this mask will have smb calculated over the entire global land surface
 <!-- MALI grids that could be used if the glc_grid logic was working. -->
 <fglcmask hgrid="0.9x1.25" glc_grid="mpas.gis20km">glc/cism/griddata/glcmaskdata_0.9x1.25_Gland5km.nc</fglcmask>  <!-- the CISM one works fine for this mesh -->
 <fglcmask hgrid="0.9x1.25" glc_grid="mpas.ais20km">lnd/clm2/griddata/glcmaskdata_0.9x1.25_60S.nc</fglcmask>
+<fglcmask hgrid="r05"      glc_grid="mpas.gis20km">lnd/clm2/griddata/glcmaskdata_0.5x0.5_everywhere.nc</fglcmask>
```

Comment:
What is the `everywhere` in this file name referring to?  I don't think we'd ever want glacier land units to be active everywhere on the LND grid if that's what it means - I think that would have a noticeable degradation on ELM performance.

Reply:
It is what it sounds like - the mask is true (ones) "everywhere". Since we're really only using the 20km GIS grid for testing and debugging, I figured this was ok (I've not noted any performance hit -- it's usually being run on only 1 or 2 nodes -- but it's certainly possible there is one). I made a few other attempts to "fix" the existing mask that was resulting in runtime errors but was unsuccessful. We can certainly do better if this seems important.

Reply:
I see - I wasn't aware the existing one was causing an error.  Activating glacier land units for the low res testing mesh seems fine.

Did you try it with `lnd/clm2/griddata/glcmaskdata_0.5x0.5_GIS_AIS.nc`?  My guess is the error comes when the GLC mesh extends beyond the masked region on the LND grid, which could be the case for the 20km GIS mesh.  I'm asking because down the road we may need to make our GIS meshes extend farther to ensure they have overlap with the OCN meshes, so if there is an extent issue with the 20km GIS mesh, it will likely show up then too.  (not necessary to figure out for this PR)